src/app.android.scss

```diff
@@ -2,8 +2,8 @@
 @import './kirby/scss/native-variables';
 
 // Import the theme’s main ruleset - both index and platform specific.
-@import '~nativescript-theme-core/scss/index';
-@import '~nativescript-theme-core/scss/platforms/index.android';
+// @import '~nativescript-theme-core/scss/index';
+// @import '~nativescript-theme-core/scss/platforms/index.android';
```

Comment:
Enten skal disse to vel bruges eller fjernes? :) Hvis der er fejl under import er det formentlig pga at deres path starter med "~". Det har før givet problemer.

Reply:
Der er en fejl i det de importerer, og Tommy er ved at rette fejlen - derfor er de indtil videre udkommenteret

Reply:
De skal bare fjernes helt - vi kommer ikke til at bruge theme-core